src/audio/src/src.c

```diff
@@ -208,6 +208,7 @@ static int src_buffer_lengths(struct comp_dev *dev, struct comp_data *cd,
 		 */
 		a->sbuf_length = 2 * nch * stage1->blk_out * r1;
 		a->sbuf_length += a->sbuf_length >> 3;
+		a->sbuf_length = ALIGN_UP(a->sbuf_length, 8);
```

Comment:
Please add a comment here to explain why this is needed?

Reply:
@iuliana-prodan good point. added a comment to better explain this.

Reply:
@singalsu @andrula-song it seems like we need some arch/platform specific config here as this will hit on differently on each target and this is generic C code. e.g.
```c
#define ARCH_SIMD_DATA_ALIGN    8
```

Reply:
The src_config.h should for HiFiX or other SIMD implementations return the buffer length align constant. We can add it later after merge of this. I assume this fixes the bug that was reported by Iuliana.

Reply:
@singalsu yes this fixes the bug reported by @iuliana-prodan.

Reply:
Ok, lets follow up as an improvement PR.